app/Models/FedCap.php

```diff
@@ -60,6 +60,19 @@ public function getRemainingCapAttribute() {
         return $this->total_attestations - $total;
     }
 
+    public function getRemainingReservedGraduateCapAttribute() {
+        $total = 0;
+        foreach ($this->caps as $cap) {
+            $total += $cap->total_reserved_graduate_attestations;
+        }
+
+        if ($total < 0) {
```

Comment:
why would this ever be true? This should never happen and whatever logic anywhere else that would make total_reserved_graduate_attestations in the negative should be fixed.